src/components/ResourceOverview/UserOverviewPage.tsx

```diff
@@ -0,0 +1,304 @@
+  const dialogStyle = dialogStyles();
+
+  const cells = members.map((member: User) => {
+    return (
+      <TableRow key={member.id}>
+        <TableCell classes={{ root: dRow.root }}>{member.name}</TableCell>
+        <TableCell classes={{ root: dRow.root }}>{member.email}</TableCell>
+        <TableCell classes={{ root: dRow.root }}>
+          {member.accessLevel
+            ? member.accessLevel.charAt(0).toUpperCase() +
+              member.accessLevel.substring(1).toLowerCase()
+            : null}
+        </TableCell>
+        <TableCell classes={{ root: optionBtn.root }}>
+          <IconButton data-id={member.id} onClick={handleClickOptions}>
+            <MoreHorizIcon style={{ color: Colours.Black }} />
+          </IconButton>
+        </TableCell>
+      </TableRow>
+    );
+  });
+
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h5">Team Member Overview</Typography>
+      <Typography variant="body2" classes={{ body2: paraStyle.body2 }}>
+        A list of all team members that can be added to an event.
+      </Typography>
+
+      <TableContainer>
+        <Table classes={{ root: table.root }}>
+          <TableHead>
+            <TableRow>
+              <TableCell classes={{ root: hRow.root }}>Name</TableCell>
+              <TableCell classes={{ root: hRow.root }}>Email</TableCell>
+              <TableCell classes={{ root: hRow.root }}>Role</TableCell>
+              <TableCell classes={{ root: optionBtn.root }} />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {cells}
+            <Popper
+              id={id}
+              open={open}
+              popperOptions={{
+                modifiers: { offset: { enabled: true, offset: '-69.5,0' } },
+              }}
+              anchorEl={anchorEl}
+            >
+              <div>
+                <Table className={classes.tablePopper}>
+                  <TableBody>
+                    <TableRow
+                      hover
+                      classes={{ hover: optionBtn.menuHover }}
+                      onClick={handleClickEdit}
+                    >
+                      <TableCell classes={{ root: optionBtn.menuCell }}>
+                        <Typography variant="body2">Edit</Typography>
+                      </TableCell>
+                    </TableRow>
+                    <TableRow hover onClick={() => setOpenModal(true)}>
+                      <TableCell classes={{ root: optionBtn.menuDelete }}>
+                        <Typography variant="body2">Delete</Typography>
+                      </TableCell>
+                    </TableRow>
+                  </TableBody>
+                </Table>
+              </div>
+            </Popper>
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <Dialog classes={{ paper: dialogStyle.paper }} open={openModal}>
+        <DialogTitle classes={{ root: dialogStyle.dialogTitle }}>
+          <Typography variant="h6">
```

Comment:
I think this is causing the warning in the console `Warning: validateDOMNesting(...): <h6> cannot appear as a child of <h2>.` Either don't use DialogTitle or see if you can change DialogTitle to use the "h6" variant.